jellyfin_kodi/jellyfin/ws_client.py

```diff
@@ -54,24 +55,44 @@ def run(self):
         LOG.info("Websocket url: %s", wsc_url)
```

Comment:
I reqlize this isn't really related to this PR, but I had to note it down somewhere when I spotted it;
The `server.replace` above is problematic if the websocket url contains `https?` anywhere else than in the protocol.
It would be better to `server.split(':', 1)` and join again after doing the protocol swap (if swapping protocol is even required for the websocket library)

Reply:
Hmm, it is possible.  Unlikely, but it would be possible for the replace to hit multiple instances.
```python
server = server.replace('https', "wss") if server.startswith('https') else server.replace('http', "ws")
```

And yes, it is required to change the protocol for the websockets library.  Also yes, that line is mixing single and double quotes, and is therefore disgusting.  A `.split(:)` also isn't a good solution though, because if somebody is accessing their server over IP:Port it will break.  So probably keep the replace, but change it to `server.replace('https://', 'wss://')` instead.  If _that_ ever hits multiple instances in a single URL, somebody deserves to get throat punched.

Reply:
That's why I specified `.split(':', 1)`, which performs a single split from the left hand side

Reply:
I don't like `.split()`.  It still leaves you with a list that you then have to pull the correct index of.  Minor, but still another step.  I targeted the replace to include the `://` instead